### PR TITLE
[SDK-2464] Add guidance for org validation

### PR DESCRIPTION
**Pull request by evansims**
This PR updates the README to include additional information about Organizations validation and adds an example of how an application can handle this logic itself where a customer needs more flexibility, such as a multi-tenant/organization scenario.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +199,67 @@ if ($invite = $auth0->getInvitationParameters()) {
 
 After successful authentication via the Universal Login Page, the user will arrive back at your application using your configured `redirect_uri`, their token will be automatically validated, and the user will have an authenticated session. Use `getUser()` to retrieve details about the authenticated user.
 
+#### Validation guidance
+
+In the examples above, our application is operating with a single, configured Organization. In this case, the SDK automatically handles token validation for you: it will ensure the user's JWT has an `org_id` claim and that it matches.
+
+However, if your application requires more flexibility (such as an API supporting multiple tenants, and therefore multiple organizations) then your application should validate the `org_id` claim itself to ensure the value received is expected and known.
```
Comment (stevehobbsdev) [inline README.md:206]: What we're describing here isn't necessarily about flexibility, it's more to do with security. In cases where the app doesn't know what Organization it is to log into (and thus doesn't send a org id to the Auth0 endpoint) but we get an org id claim in the token, we can't validate it in the SDK. So the application must validate it to make sure that the org id value is expected based on the data the application has about its customers. 

Make sense? The wording we came up with internally should help clarify this point.

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Looks good, just had a comment about some wording.

File: README.md (hunk 1 above)
Comment (evansims) [reply README.md]: @stevehobbsdev I've made some wording tweaks, see what you think

**Review by stevehobbsdev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stevehobbsdev) [reply README.md]: Better :) thanks

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**